changelog.yaml

```diff
@@ -18,7 +18,15 @@
 # Do not change the order ("NEXT" needs to be on top).
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: internal
+    impact: patch
+    title: Fix clustertask
+    description: >-
+      Fix variables in clustertask to changed structure.
+    warning:
+    upgradeNotes: See upgrade notes of version 0.5.0

```

Comment:
If you have 0.5.0 already installed, do you need to consider anything? If not, then just leave this field empty here.

Reply:
Done: https://github.com/SAP/stewardci-core/pull/173/commits/9a3b54b1ee45f6e71824f04398876cd2758ad922